invenio_communities/assets/semantic-ui/js/invenio_communities/members/components/SearchBarWithFilters.js

```diff
@@ -30,65 +30,70 @@ export class SearchBarWithFilters extends Component {
     const currentSort = currentQueryState.sortBy;
     const filters = currentResultsState.data.aggregations; // TODO: filters come from the backend
     return (
-      <Grid>
-        <Grid.Row>
-          <Grid.Column className="flex-container" width={16}>
-            <SearchBar
-              uiProps={{ className: "contents-container" }}
-              placeholder={searchBarPlaceholder}
-            />
+      <>
+        <div className="auto-column-grid">
```

Comment:
Why do we remove the `Grid` cmp of SUI in favour of divs with custom `className`?

Reply:
because grid with added custom flex display is not intuitive. Ideally I would prefer to have an UI component to handle this but it would require some more work, so I went for this solution to have automatically adjusted searchbar length

Reply:
maybe add a comment to explain the why and avoid that someone (e.g. me in the future :)) will refactor it to use `Grid`?